src/app/app.component.html

```diff
@@ -1,940 +1,28 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * The content below * * * * * * * * * * * -->
-<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * -->
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * Delete the template below * * * * * * * * * * -->
-<!-- * * * * * * * to get started with your project! * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+<body>
```

Comment:
I'm surprised this is working. Anyhow, the structure is:

```
<html>
<head></head>
<body>
<app-component></app-component>
</body>
</html>
```

You've basically added `body`  tag into `body`. I don't think that's correct.

Reply:
I was a bit confused about the file structure at first, so I added that in the appcomponent again and then realized it's already in the index.html file so I deleted it but left there the body. When I think about it now I'm quite surprised too. It's fixed.